### DISCUSSION TITLE
Please remove spammer prevention

### DISCUSSION THREAD

**Discussion by IveMalfunctioned** [GriefPrevention/GriefPrevention#1710] [Ideas]
This is GriefPrevention, not SpammerPrevention. 
The "moderation" just gets on my nerves. Like, banning spammers is enabled by default? And it perm bans them instead of temp mutes? Everyone just disables it anyways.

**Comment by Jikoo** [comment]
You can already disable the spam protection feature by setting `GriefPrevention.Spam.Enabled` to `false`.

**Comment by IveMalfunctioned** [comment]
Can it be disabled by default? Only because it bans spammers.

**Reply by Jikoo** [reply]
While personally I don't hold with the ban for spam, that is a fully configurable feature that is pretty explicitly labelled inside the spam settings. You're unlikely to see a change to the defaults until the entirety of spam handling gets rewritten, which is slated to be done Eventually(TM).

**Comment by Blzd87** [comment]
I keep getting people banned for spam while they are creating chest shops due to them writting in chat the amount to sell for and if its the same price then the antispam thinks they are being bad and bans them... so it would be good if there was a hook in with QuickShop plugin as the players typing the price does not get posted into chat its just for the shop plugin to get the price for the shop creation. 

if that is even possible... as for now i just unban them when they let me know. but would be nice if this could be added as en exemption for QuickShop and Grief Prevention?

**Comment by IveMalfunctioned** [comment]
Or you could just... disable GriefPrevention's anticheat
